core/modules/parser/QSMySqlParser.g4

```diff
@@ -4,6 +4,14 @@ import MySqlParser;
 
 options { tokenVocab=QSMySqlLexer; }
 
+// adds `MINUS?` before REAL_LITERAL
+constant
```

Comment:
I don't see the storm used in the rest of the gramma. Who's using it?
Also, there is one use of 'constants' in the gramma. Are these ('constant' and 'constants') related?

Reply:
`constants` uses `constant`. `constants` is used plenty in MySqlParser.g4

Reply:
e.g. https://github.com/lsst/qserv/blob/master/core/modules/parser/MySqlParser.g4#L723

Reply:
I see it! For some reason GitHub didn't show me the whole file, so I just missed those.

Reply:
it's a different file :-)
overrides for that file are in this file.

Reply:
Thanks!